app/views/static_pages/donate.html.erb

```diff
@@ -1,20 +1 @@
-<section class="pt-5 pb-5" id="donate">
```

Comment:
Per the above comment, this sets the styling and markup appropriate for the `/donate` page. I think we should keep this here, and make the _donate_content partial work for this view and the adopter_fosterers/donations view.

Reply:
Done the Changes